Examples/Framework/include/ActsExamples/Framework/Sequencer.hpp

```diff
@@ -78,6 +78,8 @@ class Sequencer {
     std::string outputDir;
     /// output name of the timing file
     std::string outputTimingFile = "timing.tsv";
+    // Enable or disable detailed per event timing writing
+    bool enableEventTiming = false;
```

Comment:
I think this should be steered with an output file path which can be present or not. Similar to `outputTimingFile`

Reply:
I updated it, is it what you had in mind?